content/docs/guides/continuous-delivery/google-cloud-build.md

```diff
@@ -51,7 +51,7 @@ Set the appropriate `gcp:project` and `gcp:region` values. For example, if your
 
 `pulumi config set gcp:project awesome-project` and `pulumi config set gcp:region us-central`
 
-**Note**: Not all resources are available in all regions. Please check that GCP supports the region and resource combination [here](https://cloud.google.com/about/locations/).
+**Note**: Not all resources are available in all regions. Check that GCP supports the region and resource combination. [Cloud locations](https://cloud.google.com/about/locations/).
```

Comment:
i was matching the link text to the heading on that page.. but i think i dont understand what ur suggesting?

Reply:
Ah, sorry. I should've just used the Suggestion thing here. Here's what I mean:

```suggestion
**Note**: Not all resources are available in all regions. Check that GCP supports [the region and resource combination](https://cloud.google.com/about/locations/).
```

... only because that's what we want to suggest they use the page at that URL to verify. Figured that would be clearer than just "Cloud locations". Defer to you, though.